src/ControlSystem/ControlErrors/Size/ComovingCharSpeedDerivative.hpp

```diff
@@ -0,0 +1,415 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class DataVector;
+namespace Frame {
+struct Distorted;
+struct Grid;
+}  // namespace Frame
+/// \endcond
+
+namespace control_system::size {
+
+/*!
+ * \brief Computes the derivative of the comoving characteristic speed
+ * with respect to the size map parameter.
+ *
+ * \param result the derivative of the comoving char speed
+ *        \f$d v_c/d\lambda_{00}\f$, which is computed here using
+ *        Eq. (\f$\ref{eq:result}\f$).
+ * \param lambda_00 the map parameter \f$\lambda_{00}\f$
+ * \param dt_lambda_00 the time derivative of the map parameter
+ * \param horizon_00 the average coefficient of the horizon \f$\hat{S}_{00}\f$
+ * \param dt_horizon_00 the time derivative of horizon_00
```

Comment:
Are these values the regular spherical harmonic values, or are they `Spherepack` values? We should be explicit because I've been confused by this in the past where we don't specify.